src/components/stepforms/widgets/DateComponents/DateRangeInput.vue

```diff
@@ -522,6 +525,9 @@ export default class DateRangeInput extends Vue {
   .widget-date-input__reset-button {
     color: white;
   }
+  .widget-date-input__reset-button + .widget-date-input__type-icon {
+    padding-left: 0;
+  }
```

Comment:
I'm not sure to understand what are these passing-left: 0 and padding-right:0 (below) for, and why this is only a rule when the background is colored.
Maybe a little comment will help ? Or maybe it would o to put it whether colored background or not ? Or should this be only in compact mode ?